BLE_GattServer_FOTAService/MCUboot/README.md

```diff
@@ -0,0 +1,163 @@
+# FOTA service MCUboot example
+
+This application extends the [Mock](../Mock) example by including a bootloader based on MCUboot.
```

Comment:
A link to the specification would be welcomed.

Reply:
It is linked in the mock [README](https://github.com/noonfom/mbed-os-example-ble/blob/mcuboot/BLE_GattServer_FOTAService/Mock/README.md).